### DISCUSSION TITLE
How can I close the undo/redo of the input

### DISCUSSION THREAD

**Discussion by dcc-xypc** [element-plus/element-plus#11610] [Q&A]
When there are some input boxes on the page ,we can use Ctrl+Z and Ctrl+Y to undo/redo our opration. It's a good feature, but I want to close is.How shoud I do.

**Comment by chenxch** [comment]
You can add @keydown to el-input.
[demo](https://element-plus.run/#eyJBcHAudnVlIjoiPHRlbXBsYXRlPlxuICA8ZWwtaW5wdXQgdi1tb2RlbD1cImlucHV0XCIgcGxhY2Vob2xkZXI9XCJQbGVhc2UgaW5wdXRcIiBAa2V5ZG93bj1cImhhbmRsZUtleWRvd25cIi8+XG48L3RlbXBsYXRlPlxuXG48c2NyaXB0IGxhbmc9XCJ0c1wiIHNldHVwPlxuaW1wb3J0IHsgcmVmIH0gZnJvbSAndnVlJ1xuY29uc3QgaW5wdXQgPSByZWYoJycpXG5cblxuZnVuY3Rpb24gaGFuZGxlS2V5ZG93bihlKXtcbiAgY29uc29sZS5sb2coZSlcbiAgaWYoZS5jdHJsS2V5ICYmIFsneicsICd5J10uaW5jbHVkZXMoZS5rZXkpKXtcbiAgICBlLnByZXZlbnREZWZhdWx0KClcbiAgICByZXR1cm4gZmFsc2VcbiAgfVxufVxuPC9zY3JpcHQ+XG4iLCJpbXBvcnRfbWFwLmpzb24iOiJ7XG4gIFwiaW1wb3J0c1wiOiB7fVxufSIsIl9vIjp7fX0=)

**Reply by chenxch** [reply]
@dcc-xypc 是否还有其他问题哈
